Doc/library/__main__.rst

```diff
@@ -155,7 +155,7 @@ error-prone as other functions within the module could be unintentionally using
 the global variable instead of a local name.  A ``main`` function solves this
 problem.
 
-Using a ``main`` function has the added benefit of the ``echo`` function itself
+Using a ``main`` function has the added benefit of the ``main`` function itself
```

Comment:
I don't think it's a typo.

Reply:
my thought was using a `main` function makes `echo` importable without side effects, but also makes the `main` importable.